ArcGISRuntimeSDKQt_QMLSamples/EditData/UpdateAttributesFeatureService/UpdateAttributesFeatureService.qml

```diff
@@ -132,61 +130,20 @@ Rectangle {
                 }
             }
         }
-    }
 
-    // map callout window
-    Rectangle {
-        id: callout
-        width: row.width + (10 * scaleFactor) // add 10 for padding
-        height: 40 * scaleFactor
-        radius: 5
-        border {
-            color: "lightgrey"
-            width: .5
-        }
-        visible: false
 
-        MouseArea {
-            anchors.fill: parent
-            onClicked: mouse.accepted = true
+        calloutData {
+            title: "<br><b><font size=\"+2\">" + damageType + "</font></b>"
+            location: calloutLocation
```

Comment:
if the above binding works, you can probably do it here and then you wouldn't need the `calloutLocation` property at all...